sections/futures/TradeCrossMargin/TradeCrossMargin.tsx

```diff
@@ -105,13 +105,15 @@ export default function TradeCrossMargin({ isMobile }: Props) {
 						onChange={(index: number) => {
 							const type = CROSS_MARGIN_ORDER_TYPES[index];
 							setOrderType(type as FuturesOrderType);
+							const decimals = suggestedDecimals(marketAssetRate);
+							const offset = 1 / 10 ** (decimals ?? 1);
 							const price =
 								(type === 'limit' && leverageSide === 'long') ||
 								(type === 'stop market' && leverageSide === 'short')
-									? floorNumber(marketAssetRate, 0)
+									? floorNumber(marketAssetRate, 2) - offset
```

Comment:
It still has `floorNumber(marketAssetRate, 2)` where it should either now be `floorNumber(marketAssetRate, decimals)` or just `floorNumber(marketAssetRate)` and let the function dictate suggested decimals. Also it should be minus the offset on floor and plus the offset on ceil.

Reply:
@avclarke my bad, have done the changes 👍🏽